oshi-core/src/main/java/oshi/util/platform/windows/PerfCounterQuery.java

```diff
@@ -164,13 +165,13 @@ private PerfCounterQuery() {
      */
     public static <T extends Enum<T>> Map<T, Long> queryValuesFromWMI(Class<T> propertyEnum, String wmiClass) {
         WmiQuery<T> query = new WmiQuery<>(wmiClass, propertyEnum);
-        WmiResult<T> result = WmiQueryHandler.createInstance().queryWMI(query);
+        WmiResult<T> result = Objects.requireNonNull(WmiQueryHandler.createInstance()).queryWMI(query);
         EnumMap<T, Long> valueMap = new EnumMap<>(propertyEnum);
         if (result.getResultCount() > 0) {
             for (T prop : propertyEnum.getEnumConstants()) {
                 switch (result.getCIMType(prop)) {
                 case Wbemcli.CIM_UINT16:
-                    valueMap.put(prop, Long.valueOf(WmiUtil.getUint16(result, prop, 0)));
+                    valueMap.put(prop, (long) WmiUtil.getUint16(result, prop, 0));
```

Comment:
I was amazed to learn the bytecode for this is identical.  I had thought that "unboxing" caused some sort of difference but not here.